### ISSUE TITLE
FPT_TST_EXT.1.1

### ISSUE THREAD

**Issue by ajcousi** [commoncriteria/operatingsystem#29]
Sadly this isn't as simple as we'd like it to be - need to discuss how to reword it to cover what actually happens.

**Comment by ajcousi** [comment]
useful resources
http://blogs.msdn.com/b/b8/archive/2011/09/22/protecting-the-pre-os-environment-with-uefi.aspx
https://technet.microsoft.com/en-us/windows/dn168167.aspx
https://msdn.microsoft.com/en-us/library/windows/hardware/dn653311(v=vs.85).aspx
https://technet.microsoft.com/en-us/library/hh824987.aspx

**Comment by jeffblank** [comment]
Removed things that were out of scope e.g. hardware/platform requirements.
